tests/EpisodeTest.php

```diff
@@ -4,36 +4,311 @@
+    }
+
+    /**
+     * Assert that admin user can see the dashboard stats.
+     *
+     * @return [type] [description]
+     */
+    public function testAdmincanSeeStats()
+    {
+        $user = factory('Suyabay\User')->create(['role_id' => 3]);
+        factory('Suyabay\Channel')->create();
+        $episodes = factory('Suyabay\Episode', 5)->create();
+
+        $this->actingAs($user)
+            ->call(
+                'GET',
+                '/dashboard'
+            );
+        $this->assertViewHas('data');
+        $this->see($episodes[0]['episode_name']);
+        $this->see($episodes[0]['episode_description']);
+        $this->see(3);
+        $this->see('Active Episodes');
+    }
+
+    /**
+     * Assert a user can see all episodes.
+     *
+     * @return void
+     */
+    public function testAdminUserCanSeeAllEpisodes()
+    {
+        $user = factory('Suyabay\User')->create(['role_id' => 3]);
+        factory('Suyabay\Channel')->create();
+        $episodes = factory('Suyabay\Episode', 5)->create();
+
+        $this->actingAs($user)
+            ->call(
+                'GET',
+                '/dashboard/episodes'
+            );
+        $this->see($episodes[0]['episode_name']);
+        $this->see($episodes[0]['episode_description']);
+
+        $this->assertViewHas('episodes');
+    }
+
+    /**
+     * assert that an admin user can create a new episode.
+     *
+     * @return void
+     */
+    public function testAdminCanCreateEpisodes()
+    {
+        $user = factory('Suyabay\User')->create(['role_id' => 3]);
+        factory('Suyabay\Channel')->create();
+
+        $this->actingAs($user)
+             ->call(
+                 'GET',
+                 '/dashboard/episode/create'
+             );
+        $this->see('Create Episode');
+        $this->assertViewHas('channels');
     }
 
+    /**
+     * Assert admin can edit an episode.
+     *
+     * @return void
+     */
+    public function testAdminCanEditEpisode()
+    {
+        $user = factory('Suyabay\User')->create(['role_id' => 3]);
+        factory('Suyabay\Channel')->create();
+        $episodes = factory('Suyabay\Episode', 5)->create();
+
+        $this->actingAs($user)
+            ->call(
+                'GET',
+                '/dashboard/episode/1/edit'
+            );
+            $this->assertViewHasAll(['episode', 'channels']);
+        $this->actingAS($user)
+            ->call(
+                'PUT',
+                '/dashboard/episode/edit',
+                [
+                    'episode' => 'Swanky updated name',
+                    'description' => 'Swanky updated description',
+                    'episode_id' => 1,
+                    'channel_id' => 1
+                ]
+            );
+        $this->call('GET', '/episodes');
+        $this->assertViewHas('episodes');
+    }
+
+    /**
+     * Assert that an admin can activate an episode.
+     *
+     * @return void
+     */
+    public function testAdminCaActivateEpisode()
```

Comment:
**Code Quality - Nitpick**

I believe the name of this method should be `testAdminCanActivateEpisode` instead of testAdminCaActivateEpisode`